core/code/layerchooser.js

```diff
@@ -253,13 +257,7 @@ window.addLayerGroup = function (name, layerGroup, defaultDisplay) {
   window.layerChooser.addOverlay(layerGroup, name);
 };
 
+// !!deprecated: use `layerChooser.removeLayer` directly (https://leafletjs.com/reference.html#control-layers-removelayer)
```

Comment:
leaflet doc is confusing since `layerChooser.removeLayer` removes the layer from the map by default

Reply:
But a line below you can see clear usage sample that clarifies any confusion)